keadm/cmd/keadm/app/cmd/common/constant.go

```diff
@@ -78,4 +78,8 @@ const (
 
 	// CGroupDriver is type of edgecore Cgroup
 	CGroupDriver = "cgroupdriver"
+
+	// PackagePath sets the storage path of the kubeedge installation packages
+	// eg: "kubeedge-v1.4.0-linux-amd64", "kubeedge-v1.4.0-linux-amd64.tar.gz" )
```

Comment:
eg is weird.
```suggestion
	// eg: "/etc/kubeedge/kubeedge-v1.4.0-linux-amd64.tar.gz" )
```

Reply:
u express more clearly

Reply:
u express more clearly..

Reply:
Need to consider the case that users specify a dir which cover all unziped files, which means this option should support following 2 ways:
- /path/to/release_tar
- /path/to/dir_unziped_from_release